pinot-plugins/pinot-stream-ingestion/pinot-kafka-2.0/pom.xml

```diff
@@ -35,7 +35,7 @@
   <url>https://pinot.apache.org/</url>
   <properties>
     <pinot.root>${basedir}/../../..</pinot.root>
-    <kafka.lib.version>2.0.0</kafka.lib.version>
+    <kafka.lib.version>2.8.1</kafka.lib.version>
```

Comment:
i dunno if it is worthy to change the artifactId to `pinot-kafka-2.8` or `pinot-kafka-2.x`, but it might be confusing going forward.

Reply:
For backward compatibility, let's keep it as 2.0 and mention that this is for 2.x versions.